tests/subsys/spm/thread_swap/src/main.c

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <ztest.h>
+#include <secure_services.h>
+#include <kernel.h>
+
+static struct k_delayed_work interrupting_work;
+static volatile bool work_done;
+
+static void work_func(struct k_work *work)
+{
+	work_done = true;
+	spm_busy_wait(20000);
+}
+
+void test_spm_service_thread_swap1(void)
+{
+	int err;
```

Comment:
A comment on what is being tested would be good, currently you have to read the commit message to see a description of what this test does and why its needed.

Reply:
Done, take a look